stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.27.0
+version: 1.27.1
```

Comment:
This would add a feature, so please increment the minor component of the version.